org.OpenT2T.Onboarding.WinkHub/v2/winkonboarding.js

```diff
@@ -0,0 +1,140 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+
+'use strict';
+var https = require('https');
+var inquirer = require('inquirer');
+var q = require('q');
+var hubDevice = require('./WinkHub');
+var accessTokenInfo = require('../../org.OpenT2T.Onboarding.Hub/opent2tonboarding');
+
+// module exports, implementing the schema
+module.exports = {
+
+    //
+    // question: Do we need to remove validate function?
+    // This is tied with 'inquirer' syntax, which helps validate at commandline as well.
+    //
+    onboardingFlow : [
+        {
+            action: 'get-user-input',

```

Comment:
Sure. Should be good to have 'separation of concerns'

---
In reply to: [74331940](https://github.com/openT2T/onboarding/pull/10#discussion_r74331940) [](ancestors = 74331940)